compsets/benchmark_cold_ccpp.input

```diff
@@ -0,0 +1,62 @@
+########################################################################
+
+test cpld_fv3_ccpp_384_mom6_cice_cold_bmark: fv3_ccpp_mom6_cice.exe {
+    use cpld_fv3_ccpp_mom6_cice_cold_atm_flux
+    use plat%default_cpl_benchmark
+
+    ATMRES='384'
+    TEST_DESCR="Fully coupled FV3-MOM6-CICE system - cold 384"
+ 
+    DT_ATMOS='450'
+    NPX='385'
+    NPY='385'
+    IMO='1536'
+    JMO='768'
+    FNALBC="'global_snowfree_albedo.bosu.t766.1536.768.rg.grb',"
+    FNVETC="'global_vegtype.igbp.t766.1536.768.rg.grb',"
+    FNSOTC="'global_soiltype.statsgo.t766.1536.768.rg.grb',"
+    FNSMCC="'global_soilmgldas.statsgo.t766.1536.768.grb',"
+    FNABSC="'global_mxsnoalb.uariz.t766.1536.768.rg.grb',"
+
+    FV3_mosaic="C@[ATMRES]"
+
+    # inputs and fixed files for benchmark runs to match workflow
+      INPUT_NML="input.benchmark_ccpp.nml.IN"
+     SUITE_NAME='FV3_GFS_v15p2_coupled'
+    FIELD_TABLE="field_table.gfdlmp"
+     OZONE_data="@[FV3_input_data]/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77"
+       CO2_data="@[plat%INPUTS]/FV3_input_bm2/INPUT"
+
+    # - set date YYYYMMDDHH
+    SYEAR='2012'
+    SMONTH='01'
+    SDAY='01'
+    SHOUR='00'
+    CDATE="@[SYEAR]@[SMONTH]@[SDAY]@[SHOUR]"
+
+    # - set run type for CICE in ice_in
+    RUNID='cpcice'
+
+    # - set initialization type for MOM6 
+    MOM6_RESTART_SETTING = 'r'
+
+    # set locations of ICs
+         BM_IC="/scratch1/NCEPDEV/nems/Bin.Li/S2S/FROM_HPSS/@[CDATE]"
```

Comment:
Generally we don't allow personal directory in the repository code, maybe we can change this later. E.g. get run directory $RUNDIR first , then define BM_IC=$RUNDIR/FROM_HPSS/@[CDATE]

Reply:
We're working on moving all the ICs to the climpara area and we could point to that in the date suite/benchmark tests (this location).   I don't know if you want to add all the ICs to the RT area for the datesuite test area though.  For the regression test, I added the benchmark IC files to the regression test baseline area: 
https://github.com/ufs-community/ufs-s2s-model/blob/develop/compsets/benchmarkRT_cold.input#L45  
         BM_IC="@[plat%INPUTS]/BM_IC/@[CDATE]"

Maybe we should add the 2012010100 and 2012010100 IC dates there for what are actually regression tests?

Reply:
I'd prefer all the ICs to be the RT area as we consider the RT could be independent. We do the same for the fix files in weather. It will be easier for code porting and testing as we don't need to have climpara ready to do RT.

Reply:
The ICs for 2012010100 and 2012070100 will be in the RT area in the next ufs-s2s-model update.

Reply:
It's 4TB worth of data, so I don't think you want all of the ICs in the RT area.  I would think the 3 tests that are regression tests or at most the 8 ICs currently set up in the datesuite tests?  @DeniseWorthen if we point to the RT baseline area for those 8 datesuite tests, are we then loosing the ability to run any date which I think was part of your desire when you originally made these tests?

Reply:
Yes. The datesuites would still need to point to the original data location. It should work to add a line to the datesuite tests re-setting the BM_IC location w/in each datesuite test (both cold and warm):

    # - set date YYYYMMDDHH
    SYEAR='2012'
    SMONTH='01'
    SDAY='01'
    SHOUR='00'
    CDATE="@[SYEAR]@[SMONTH]@[SDAY]@[SHOUR]"

    # set locations of ICs
    BM_IC="/scratch1/NCEPDEV/nems/Bin.Li/S2S/FROM_HPSS/@[CDATE]"